### PR TITLE
Remove unused UI elements

### LINKED ISSUE #47: Remove unused UI elements to prevent confusion
The UI currently contains some elements that either should not be included, are not in use. These either should be removed or have some have better place-holder functionality.

This includes
- Load demo
- Help
- ~'fake user icon' +~ Settings
- Privacy Policy
- Terms & Conditions
- Session status block

### PR DESCRIPTION
**Pull request by ermannolocascio**
Fixes #47. i.e. Remove unused ui items, adds template to the help page. Defines privacy policy.

### CODE REVIEW

File: help.php [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+                border-radius: 4px;
+                cursor: pointer;
+            }
+            .copy-button:hover {
+                background-color: #0056b3;
+            }
+            .copy-icon {
+                margin-left: 5px;
+            }
+            </style>
```
Comment (sjvrijn) [inline help.php:37]: Can we put this in a css file instead?

File: help.php [hunk 2]
```diff
@@ -12,19 +45,48 @@
...
-                                class, the side navigation will take on a light color scheme. The
-                                <code>.sb-sidenav-dark</code>
-                                is also available for a darker option.
+                            <h1>Welcome to INA TOOL</h1>
+                                <h3>Dashboard Overview</h3>
+                                <ul>This is the central hub where you can view the istitutional statements visualization and manage your data.</ul>
+
+                                <h3>Upload your File & Template</h3>
+                                <ul>
+                                You can upload only <i>.csv</i> or <i>.txt</i> files. Please, use this <a href="Institutional_statement_template - INA.txt" download>template</a>. Alternatively, the file must respect the columns ordering and naming described below:
```
Comment (sjvrijn) [inline help.php:54]: Shall we link to the [excel template](https://nlesc.sharepoint.com/:x:/s/esi-far77/Eeuypd5ADlhFuj6bqazIt5IB4Yxy1of0iiIVM-yYt7Akvw?e=BXtoMa_) instead? It has helpful conditional formatting to indicate which fields are required. We should explicitly say to export/download as CSV for now, to make sure they don't try to upload an excel file instead.

File: help.php [hunk 3]
```diff
@@ -12,19 +45,48 @@
...
+                                </ul>
+
+                                <h3>Install a Local Version of the Software</h3>
+                                <ul>
+                                Please, visit the <a href='https://github.com/ESI-FAR/INA-tool' target='blank'>github</a> repository and follow the installation instructions.
+                                </ul>
+
+                                <h3>Support and Discussion</h3>
+                                <ul>
+                                    If you encounter issues not covered here, or you want to share some ideas, please use the <a href='https://github.com/ESI-FAR/INA-tool/discussions/landing'>github</a> repository.
```
Comment (sjvrijn) [inline help.php:81]: Suggestion:
```html
If you encounter issues not covered here, or you want to share some ideas,
please open an <a href='https://github.com/ESI-FAR/INA-tool/issues/new'>issue</a>
or start a <a href='https://github.com/ESI-FAR/INA-tool/discussions/landing'>discussion</a>
on the GitHub repository
```

File: privacy-policy.php [hunk 4]
```diff
@@ -12,18 +12,32 @@
...
+                            </ul>
+
+                            <h2>Third-Party Services</h2>
+                            <ul>Our website does not utilize any third-party services that track users or collect personal information.</ul>
+
+                            <h2>Changes to This Policy</h2>
+                            <ul>We may update our Privacy Policy from time to time. Any changes will be reflected on this page.</ul>
+
+                            <h2>Contact Us</h2>
+                            <ul>If you have any questions about this Privacy Policy, please contact the software contributors via the <a href="https://github.com/ESI-FAR/INA-tool" target='blank'>github</a> page.</ul>
```
Comment (sjvrijn) [inline privacy-policy.php:35]: github -> GitHub

Comment (sjvrijn) [inline privacy-policy.php:35]: github -> GitHub

File: help.php [hunk 5]
```diff
@@ -12,19 +45,48 @@
...
+                                    2. From the context menu select <i>Draw connection</i>;<br>
+                                    3. Left click on the destination shape.
+                                    </li>
+                                    <li><strong>Table Filtering:</strong> Write text in the search box of the table to focus on specific data subsets.</li>
+                                    <li><strong>Export:</strong> Download your webview as PNG image by clicking the <i><i class="fa-solid fa-file-arrow-down"></i> download PNG</i> button at the top right side on the canvas.</li>
+                                </ul>
+
+                                <h3>Install a Local Version of the Software</h3>
+                                <ul>
+                                Please, visit the <a href='https://github.com/ESI-FAR/INA-tool' target='blank'>github</a> repository and follow the installation instructions.
```
Comment (sjvrijn) [inline help.php:76]: Suggestion:

```html
To install a local version of this application, please follow the instructions listed in the README
of the <a href='https://github.com/ESI-FAR/INA-tool' target='blank'>GitHub</a> repository.
```

**Review by sjvrijn — CHANGES_REQUESTED** [review]
Thanks for cleaning this all up! I have a few suggestions about e.g. the linked template file and  phrasing of some sentences on the help page.

File: help.php (hunk 1 above)
Comment (ermannolocascio) [reply help.php:37]: You are right. I forgot to move it ;) Thanks

File: help.php (hunk 2 above)
Comment (ermannolocascio) [reply help.php:54]: Yes!

**Comment by ermannolocascio** [comment]
Just added a final commit. Plz check and approve eventually.

**Review by sjvrijn — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**